docs/changes.txt

```diff
@@ -172,6 +172,11 @@ Changes in behaviour which may result in build errors
 - wxGTK wxDirButton::Create() doesn't have unused "wildcard" parameter any
   longer, please just remove it from your code if you used it.
 
+- Several functions now take wxReadOnlyDC instead of wxDC as argument. While
```

Comment:
Some possible name alternatives, using other words instead of `ReadOnly`: `Accessor`, `Access`, `Probe`, `Inquiry`, `Examine`, `Inspect`

Reply:
Some of those (e.g. `Inspect`) seem too close to `wxInfoDC`, which is something different, and none of them seems obviously better than `ReadOnly` IMHO. But let me know if anybody really loves one of those (or something else), we can still rename it before merging...